### DISCUSSION TITLE
How to add Contact Details of Client

### DISCUSSION THREAD

**Discussion by adarshmadrecha** [madrecha/jamku#612] [QA] [answered]
Suggested by `Suryapawan`

I want to store mobile number of Client in Jamku.

**Comment by Divwork92** [comment, marked as answer]
You can add contact details of the client in 2 ways 

(A) From client view page
1. click on the particular client
2. go to profile tab
3. select confidential as private or public 
4. select profile type, add attribute, information
![image](https://user-images.githubusercontent.com/87230676/126495226-8785bb9c-3ea3-40eb-88fd-bb9afcbf5617.png)
5. click on create

(B) From profile list
1. Go to profile list page
2. click on '+ADD'
3. select the client, confidential, profile type, add attribute, information
![image](https://user-images.githubusercontent.com/87230676/126496043-1e54e130-40ae-479f-af3c-4714d4ea4e3d.png)
5. click on create

*Private - visible only to members who have the permission 
*Public - visible to all

You can even add by importing
